sisyphus/global_settings.py

```diff
@@ -58,46 +59,33 @@ def worker_wrapper(job, task_name, call):
     return call
 
 
-def update_engine_rqmt(initial_rqmt, last_usage):
-    """ Update requirements after a job got interrupted.
+def update_engine_rqmt(last_rqmt: Dict, last_usage: Dict):
+    """ Update requirements after a job got interrupted, double limits if needed
 
-    :param dict[str] initial_rqmt: requirements that are requested first
+    :param dict[str] last_rqmt: requirements that are requested first
```

Comment:
Doc string should be something like: requirements requested for the previous run (possibly too little time or memory)